wayang-applications/pom.xml

```diff
@@ -0,0 +1,145 @@
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.wayang</groupId>
+        <artifactId>wayang</artifactId>
+        <version>0.7.1</version>
+    </parent>
+
+    <artifactId>wayang-applications</artifactId>
+    <packaging>jar</packaging>
+
+    <name>wayang-applications</name>
+    <url>http://maven.apache.org</url>
+
+
+    <properties>
+        <DATA_REPO_001>file://${project.basedir}/wayang-applications/data/case-study/DATA_REPO_001</DATA_REPO_001>
+        <DATA_REPO_002>https://kamir.solidcommunity.net/public/ecolytiq-sustainability-profile</DATA_REPO_002>
+        <DIST_WAYANG_HOME>${project.basedir}/../wayang-assembly/target/apache-wayang-assembly-0.7.1-incubating-dist/wayang-0.7.1</DIST_WAYANG_HOME>
+        <WAYANG_VERSION>0.7.1</WAYANG_VERSION>
+
+        <spark.version>3.5.0</spark.version>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+
+        <!-- We also specify the file encoding of our source files, to avoid a warning -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <assertj.version>3.17.2</assertj.version>
+        <commons-io.version>2.5</commons-io.version>
+        <guava.version>19.0</guava.version>
+        <hamcrest.version>1.3</hamcrest.version>
+        <jackson.version>2.10.2</jackson.version>
+        <jacoco.version>0.8.5</jacoco.version>
+        <jodatime.version>2.10.6</jodatime.version>
+        <jsonpath.version>2.4.0</jsonpath.version>
+        <junit5.version>5.6.1</junit5.version>
+        <mockito.version>3.5.10</mockito.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.wayang</groupId>
+            <artifactId>wayang-core</artifactId>
+            <version>${WAYANG_VERSION}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.wayang</groupId>
+            <artifactId>wayang-basic</artifactId>
+            <version>${WAYANG_VERSION}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.wayang</groupId>
+            <artifactId>wayang-java</artifactId>
+            <version>${WAYANG_VERSION}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.wayang</groupId>
+            <artifactId>wayang-spark_2.12</artifactId>
+            <version>${WAYANG_VERSION}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.wayang</groupId>
+            <artifactId>wayang-api-scala-java_2.12</artifactId>
```

Comment:
@kamir the modules with ${name}_2.12 don't exist anymore. We got rid of the 2.12 as we no longer support Scala 2.11, so this would load a very old snapshot of this module. It is very likely, that this is why you can't use your newly build Kafka operators in the JavaPlanBuilder.

Reply:
This fixed the issue I struggled with.
Thanks Juri.